src/services/userService.ts

```diff
@@ -13,21 +13,22 @@ export class UserService implements IUserService {
         private readonly alertHelper: IAlertHelper
     ){}
 
-    async createUser(user: User): Promise<void> {
+    async createUser(user: User): Promise<User | undefined> {
         try {
             user.color = defineRandomColor()
             await this.userDatabaseRepository.create(user)
             this.alertHelper.alertSucess("Usuário criado com sucesso!")
+            return user
```

Comment:
I've added this user return so we could have access to the user color on context after the first login